alpha/lib/enums/UserEntryType.php

```diff
@@ -0,0 +1,9 @@
+<?php
+/**
+ * @package Core
+ * @subpackage model.enum
+ */
+interface userEntryType extends BaseEnum
+{
+	const KALTURA_QUIZ_USER_ENTRY =1;
```

Comment:
This enum should be empty and the Quiz value should be added by the Quiz plugin.